addons/efs-provisioner/2.4.x/efs-1.yaml

```diff
@@ -0,0 +1,43 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: efs-provisioner
+  labels:
+    kubeaddons.mesosphere.io/name: efs-provisioner
+    kubeaddons.mesosphere.io/provides: efs-provisioner
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "2.4.0-1"
+    appversion.kubeaddons.mesosphere.io/aws-efs-csi-drive: "2.4.0"
```

Comment:
it is not the `aws-efs-csi-driver`, so we should rename it to `efs-provisioner` so it matches the chart name